app/lib/action_controller/request_forgery_protection/exception_and_reset_strategy.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module ActionController
+  module RequestForgeryProtection
+    class ExceptionAndResetStrategy
+      def initialize(controller)
+        @controller = controller
+      end
+
+      def handle_unverified_request
+        @controller.reset_session
```

Comment:
should we also update the comment on https://github.com/3scale/porta/blob/5c9335b7bf1a347070524dc4250ddf5ac9563d33/app/lib/authenticated_system.rb#L25

Reply:
Done